Makefile

```
@@ -431,16 +410,27 @@ ifeq ($(CIRCLE_JOB),build-race-condition-debug-image)
 	docker start -i builder
 	docker cp builder:/go/src/github.com/stackrox/rox/bin/linux bin/
 else
-	docker run -i -e RACE -e CI -e CIRCLE_TAG -e GOTAGS -e DEBUG_BUILD --rm $(GOPATH_WD_OVERRIDES) $(LOCAL_VOLUME_ARGS) $(BUILD_IMAGE) make main-build-nodeps
+	docker run -i -e RACE -e CI -e CIRCLE_TAG -e GOTAGS -e DEBUG_BUILD --rm $(GOPATH_WD_OVERRIDES) $(LOCAL_VOLUME_ARGS) $(BUILD_IMAGE) make cli-build main-build-nodeps
+endif
+
+.PHONY: cli-build
+cli-build:
+ifndef CI
+	RACE=0 CGO_ENABLED=0 GOOS=$(HOST_OS) GOARCH=$(GOARCH) $(GOBUILD) ./roxctl
```

I believe this should be
```suggestion
	RACE=0 CGO_ENABLED=0 GOOS=linux GOARCH=$(GOARCH) $(GOBUILD) ./roxctl
```
because we want to build a `roxctl` that works inside the container, and the container is built with the host arch, but will always be linux-based.

---

It's also required to match what is copied below